tests/baselines/reference/publicGetterProtectedSetterFromThisParameter.errors.txt

```diff
@@ -0,0 +1,45 @@
+tests/cases/compiler/publicGetterProtectedSetterFromThisParameter.ts(33,7): error TS2446: Property 'q' is protected and only accessible through an instance of class 'A'. This is an instance of class 'B'.
+tests/cases/compiler/publicGetterProtectedSetterFromThisParameter.ts(34,7): error TS2446: Property 'u' is protected and only accessible through an instance of class 'A'. This is an instance of class 'B'.
+
+
+==== tests/cases/compiler/publicGetterProtectedSetterFromThisParameter.ts (2 errors) ====
+    class A {
+      get x() { return 0; }
+      protected set x(v: number) { }
+    
+      public get y() { return 0; }
+      protected set y(v: number) { }
+    }
+    
+    class B {
+      get q() { return 0; }
+      protected set q(v: number) { }
+    
+      protected get u() { return 0; }
+      protected set u(v: number) { }
+    
+      foo(this: A, a: A, b: B) {
+        // Should have no errors in this block
+        this.x = 0;
+        this.y = 0;
+        a.x = 0;
+        a.y = 0;
+        b.q = 0;
+        b.u = 0;
+      }
+    }
+    
+    function bar(this: A, a: A, b: B) {
+        this.x = 0;
+        this.y = 0;
+        a.x = 0;
+        a.y = 0;
+        // These should error
+        b.q = 0;
+          ~
+!!! error TS2446: Property 'q' is protected and only accessible through an instance of class 'A'. This is an instance of class 'B'.

```

Comment:
`q` and `u` are accessors on `B`, not `A`. This error message is wrong. Is that expected? (From the bug, it seems like it might be outside the scope of this fix.)

Reply:
Yeah, I'll file a new issue